modules/navier_stokes/doc/content/modules/navier_stokes/linear_wcnsfv.md

```diff
@@ -0,0 +1,67 @@
+# Weakly Compressible Navier Stokes using a Linear Finite Volume Implementation
+
+## Equations
+
+The linear finite volume implementation of the weakly compressible Navier Stokes equations is used
+to solve the following equations:
+
+- conservation of momentum
+- pressure-correction (see [SIMPLE.md])
+- turbulence equations
+- conservation of energy
+- conservation of advected passive scalars
+- conservation of an advected phase in a homogeneous mixture
+
+We refer the reader to the respective `Physics` pages, listed in [linear_wcnsfv.md#syntax], for the strong form of the equations.
+
+## Solver algorithm(s)
+
+For steady state simulations, you may use the [SIMPLE.md] executioner which implements the SIMPLE algorithm [!citep](patankar1983calculation).
+
+For transient simulations, you may use the [PIMPLE.md] executioner which implements the PIMPLE algorithm [!citep](greenshieldsweller2022), which is a blend of PISO inner iterations and SIMPLE outer iterations.
+
+## Discretization
+
+### General
+
+We use a linearized cell-centered finite volume discretization. We have implemented orthogonal
+gradient correction and skewness correction for face values and gradients, and thus can reach second-order accuracy in many cases.
+
+!alert note
+Triangular and tetrahedral meshes currently only achieve first order spatial convergence rates at the moment.
```

Comment:
Well in some cases they are first order in some case they are second. Depends on the BC etc. In most cases it is first order for pressure.